data/ext/bio/FormalParameter.rdfa

```diff
@@ -0,0 +1,40 @@
+<div>
+  <!--
+  # Schema.org Bioschemas extension proposal
+  * | Project: https://bioschemas.org
+  * | MailingList: https://lists.w3.org/Archives/Public/public-bioschemas/
+  * | Issues:
+  -->
+
+  <div typeof="rdfs:Class" resource="http://schema.org/FormalParameter">
```

Comment:
If a generalization covering other types also working with input/output is not possible, maybe changing the name could be considered

Reply:
I think it can be used with other things having input and output, but I don't know what domains to add there as here we only need it on `Workflow` (although it would be nice to have also on an individual step referenced from `hasPart`).

Reply:
What about the software case? As it will be a ComputationalWorkflow, pieces of software will be important there. Could you keep it in mind when describing domains, ranges and so? I know it adds work but it will be more usable even by others.